Apps/W1/ContosoCoffeeDemoDataset/app/Service/2.Master Data/CreateSvcItemDemoData.Codeunit.al

```diff
@@ -0,0 +1,146 @@
+codeunit 5104 "Create Svc Item Demo Data"
+{
+    Permissions = tabledata Item = rim,
+        tabledata "Item Unit of Measure" = rim,
+        tabledata "Resource Skill" = rim;
+
+    var
+        SvcDemoDataSetup: Record "Svc Demo Data Setup";
+        AdjustSvcDemoData: Codeunit "Adjust Svc Demo Data";
+        SvcDemoDataFiles: Codeunit "Svc Demo Data Files";
+        CreateSvcSetup: Codeunit "Create Svc Setup";
+        DoRunTriggers: Boolean;
+        PCSTok: Label 'PCS', MaxLength = 10, Comment = 'Must be the same as Standard and Eval demodata';
+        MachineDesc1Tok: Label 'S-100 Semi-Automatic', MaxLength = 100;
+        MachineDesc2Tok: Label 'S-200 Semi-Automatic', MaxLength = 100;
+        ServiceItemGroupCodeTok: Label 'COMMERCIAL', MaxLength = 10;
+
+    trigger OnRun()
+    begin
+        DoRunTriggers := true;
+        OnBeforeStartCreation(DoRunTriggers);
+        SvcDemoDataSetup.Get();
+
+        CreateItems();
+        OnAfterCreatedItems();
```

Comment:
the events should be at the end of the procedure, i.e. `OnAfterCreatedItems` should be inside and at the end of `CreateItems`

Reply:
I think these events make sense only if another master data will have dependency on created items. Otherwise the onInsertMasterData event should be enough.